### DISCUSSION TITLE
how to use  useTimestamp start with zero

### DISCUSSION THREAD

**Discussion by WingDust** [vueuse/vueuse#1971] [Q&A] [answered]
how to use  useTimestamp start with zero

**Comment by WingDust** [comment]
looks like vueuse will not have useTimestamp start with zero
because https://github.com/vueuse/vueuse/blob/61ae84fa4502db5fb89f66ff40404e493812c0b2/packages/shared/utils/is.ts#L17

**Comment by antfu** [comment, marked as answer]
```ts
useTimestamp({
  offset: -Date.now()
})
```
